specification/powerbiprivatelinks/resource-manager/Microsoft.PowerBI/stable/2020-06-01/powerbiprivatelinks.json

```diff
@@ -0,0 +1,1627 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.PowerBI/privateLinkServicesForPowerBI/operationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "PrivateLinkServiceResourceOperationResults"
+        ],
+        "operationId": "PrivateLinkServiceResourceOperationResults_Get",
+        "x-ms-examples": {
+          "Result of operation on private link resources": {
+            "$ref": "./examples/PrivateLinkServiceResourceOperationResults_Get.json"
+          }
+        },
+        "description": "Gets operation result of Private Link Service Resources for Power BI.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AsyncOperationDetail"
+            },
+            "headers": {}
+          },
+          "default": {
+            "description": "Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PowerBI/privateLinkServicesForPowerBI": {
+      "get": {
+        "tags": [
+          "PrivateLinkServices"
+        ],
+        "operationId": "PrivateLinkServices_ListByResourceGroup",
+        "x-ms-examples": {
+          "List private link resources in a resource group": {
+            "$ref": "./examples/PrivateLinkServices_ListByResourceGroup.json"
+          }
+        },
+        "description": "Gets all the private link resources for the given resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/TenantResource"
+              }
+            },
+            "headers": {}
+          },
+          "default": {
+            "description": "Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown.",
```

Comment:
>Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown. [](http://example.com/codeflow?start=28&length=120)

This goes into the docs so be sure to use customer-friendly wording.